tools/bin/hawqregister

```diff
@@ -1145,7 +1153,7 @@ def main(options, args):
 
     failure_handler = FailureHandler(conn)
     # register
-    if options.yml_config and ispartition(options.yml_config):
+    if check_file_exist(options.yml_config) and ispartition(options.yml_config):
```

Comment:
Should function check_file_exist(options.yml_config) be called in both partition table and non-partition table?